tests/unit_tests/test_data_ingestion.py

```diff
@@ -0,0 +1,61 @@
+import pickle
+import sys
+import os
+import pandas as pd
+import numpy as np
+import requests
+import tarfile
+
+sys.path.append("/home/deepak/practice_git_repo/assignment2/mle-training/")
```

Comment:
py test should not deal with any paths.... once you package this and zip it .... it goes across different systems and different platforms.... and their installation of your packages will fail.

Reply:
you can copy and create data set (can be  a subset) here and then train your model and then get the results and verify the pipeline